go.mod

```diff
@@ -5,5 +5,14 @@ go 1.18
 require (
 	github.com/jinzhu/inflection v1.0.0
 	github.com/jinzhu/now v1.1.5
-	golang.org/x/text v0.14.0
+	golang.org/x/text v0.18.0
```

Comment:
Hi, we need to ensure that gorm.io/gorm does not introduce any external dependencies.

We could put all related dependencies within the tests package.